examples/bank/bank.go

```diff
@@ -57,12 +57,19 @@ type Account struct {
 	Balance int64
 }
 
-func (a Account) encode() ([]byte, error) {
-	return json.Marshal(a)
+func (a Account) encode(retErr *error) []byte {
```

Comment:
Something was lost in the translation from the blog post - the primary change they made is embed the error on a struct which provides some other facility and move the `err != nil` from the caller to the repeatedly-invoked method.

I don't see any methods that fit that pattern particularly well in this file. The out-parameter API also makes it easier to ignore errors. I'm :-1: on this.